lib/PanelTestProvider.jsm

```diff
@@ -101,38 +102,63 @@ const MESSAGES = () => [
       body:
         "The New Enhanced Tracking Protection, gives you the best level of protection and performance. Discover how this version is the safest version of firefox ever made.",
       cta_url: "https://blog.mozilla.org/",
+      cta_type: "OPEN_URL",
     },
     targeting: `firefoxVersion > 69`,
     trigger: { id: "whatsNewPanelOpened" },
   },
   {
     id: "WHATS_NEW_70_2",
     template: "whatsnew_panel_message",
+    order: 1,
     content: {
       published_date: 1560969794394,
       title: "Another thing new in Firefox 70",
       body:
         "The New Enhanced Tracking Protection, gives you the best level of protection and performance. Discover how this version is the safest version of firefox ever made.",
       link_text: "Learn more on our blog",
       cta_url: "https://blog.mozilla.org/",
+      cta_type: "OPEN_URL",
     },
     targeting: `firefoxVersion > 69`,
     trigger: { id: "whatsNewPanelOpened" },
   },
   {
     id: "WHATS_NEW_69_1",
     template: "whatsnew_panel_message",
+    order: 1,
     content: {
       published_date: 1557346235089,
       title: "Something new in Firefox 69",
       body:
         "The New Enhanced Tracking Protection, gives you the best level of protection and performance. Discover how this version is the safest version of firefox ever made.",
       link_text: "Learn more on our blog",
       cta_url: "https://blog.mozilla.org/",
+      cta_type: "OPEN_URL",
     },
     targeting: `firefoxVersion > 68`,
     trigger: { id: "whatsNewPanelOpened" },
   },
+  {
+    id: "WHATS_NEW_70_3",
+    template: "whatsnew_panel_message",
+    order: 2,
+    content: {
+      published_date: 1560969794394,
+      layout: "tracking-protections",
+      title: { string_id: "cfr-whatsnew-tracking-blocked-title" },
+      subtitle: { string_id: "cfr-whatsnew-tracking-blocked-subtitle" },
+      icon_url:
+        "resource://activity-stream/data/content/assets/protection-report-icon.png",
+      icon_alt: "Protection Report icon",
+      body: { string_id: "cfr-whatsnew-tracking-protect-body" },
+      link_text: { string_id: "cfr-whatsnew-tracking-blocked-link-text" },
+      cta_url: "protections",
+      cta_type: "OPEN_ABOUT_PAGE",
+    },
+    targeting: `firefoxVersion > 69 && totalBlockedCount > 0`,
```

Comment:
Not sure if it's because devtools doing something, but I ended up seeing this message showing 0 blocked on a new profile (with appropriate testing prefs set). So some reason the targeting `totalBlockedCount > 0` isn't being honored ??

Reply:
The testing is confusing: [local test providers override message targeting](https://github.com/mozilla/activity-stream/blob/master/lib/PanelTestProvider.jsm#L116). This is ok when you click `Show` to force a message to appear but makes it weird for interactions that have multiple steps like What's New (force show the badge -> then open the dropdown).
I'll try to find a way to improve this but for now it's expected.